### ISSUE TITLE
[Chrome] Theming using .crx doesn't use custom colors

### ISSUE THREAD

**Issue by terencode** [adapta-project/adapta-gtk-theme#423]
**Reporter info**

 ```
 * Distribution - Arch Linux
 * Gtk+ 3.0 version - 3.22.15
 * Desktop environment - Gnome 3.25.1
 ```

**Adapta version 3.91.0.48.r2.2dc24bd5**

**Related Application and/or Shell theming (or name of widget)**
Chrome / Chromium

**Actual issue**
Whereas the regular gtk+ theme option from Chrome catches the custom colors specified at compilation time(text selection color changes for example), Adapta chrome theme do not.

**Step to reproduce**
Drag'n Drop .crx file to chrome/chromium and enable the theme.

**Expected behaviour**
When specifying custom colors at compilation time, the custom adapta chrome theme should use them.

**Comment by tista500** [comment]
Chrome(ium) version?

Basically, `crx` files do not contain the `selection-color`. See https://github.com/adapta-project/adapta-gtk-theme/blob/master/extra/chrome/adapta-theme/manifest.json

So Chrome(ium) pulls selected-item colours form Gtk+ 2x or Gtk+ 3.x/4.0 directly. Which part of Chrome(ium) theming was mismatched against your customed colours?

Regards.

**Comment by terencode** [comment]
Chromium 58.0.3029.110
Ah yes, I see.
Well when I select text, the color is different and matches my custom colors with the gtk+ but not with the crx where the selection color looks standard to me.

**Comment by tista500** [comment]
Hmm, it seems chrome(ium) might detach Gtk+ theming when users applied extra `crx` theming... Unfortunately I don't have any ideas to solve that miscolouring currently.

Today I've started extra fixes for newer libgtkui with Gtk+ 3.x/4.0 backend. So in future, we don't have to use those compressed crx files for theming (Actually Chrome(ium) 60.0.30xx trunk is able to pull Gtk+ 3.x/4.0 colour definitions from theming more nicely without any crx files).

So I set this issue as "can't fix" currently, sorry.
Regards.
